### PR TITLE
feat: One DB support

### PR DESCRIPTION
**Pull request by wsdt**
# Use 1 DB for all services

Right now the Backend service needs to be deployed per network, as well as the database. 

This is just a waste of infra budget. 

This first PR should remove the necessity to deploy a dedicated database for each service. 

Please carefully review. All Tests run through. 

--

The next PR ideally merges all services into one container to save even more infra costs.

### CODE REVIEW

**Comment by wsdt** [comment]
merging as Souradeep suggested :-)

**Review by souradeep-das — COMMENTED** [review]
LGTM!

File: src/service.ts [hunk 1]
```diff
@@ -393,18 +394,22 @@ export class LightBridgeService extends BaseService<TeleportationOptions> {
...
-      await this._putDepositInfo(chainId, latestBlock)
+      await this._putDepositInfo(depositChainId, latestBlock)
 
       // Only do on boba l2
       if (this.options.airdropConfig?.airdropEnabled) {
         await this._airdropGas(disbursement, latestBlock)
       } else {
-        this.logger.info(`Gas airdrop is disabled on chainId: ${chainId}.`)
+        this.logger.info(
+          `Gas airdrop is disabled on chainId: ${depositChainId}.`
```
[the remark this answers is not in the document]
Comment (wsdt) [reply src/service.ts:411]: no it's depositChainId (where the funds arrive -> eligible for airdrop yes/no) :-) 
serviceChainid (conceptually the origin chainid)

**Comment by wsdt** [comment]
thanks for the review! :-)

**Review by Boyuan-Chen — COMMENTED** [review]
Looks good to me

**Pull request merged**